src/pages/blog.js

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import { graphql } from "gatsby";
+
+import BlogCardDesktop from "../components/blog/blogCards/BlogCardDesktop";
+import BlogCardMobile from "../components/blog/blogCards/BlogCardMobile";
+import Categories from "../components/blog/Categories";
+import FeaturedPost from "../components/blog/FeaturedPost";
+import SearchBar from "../components/blog/SearchBar";
+
+export default function Blog({ data }) {
+  const {
+    contentfulBlogHomepage: {
+      featuredPost,
+      facialAestheticHeroes,
+      bodyAestheticHeroes,
+      // sexualEnhancementHeroes,
+    },
+  } = data;
+
+  const categories = [
+    facialAestheticHeroes,
+    bodyAestheticHeroes,
+    // sexualEnhancementHeroes,
+  ];
+
+  console.log(categories);
+
+  return (
+    <div>
+      <FeaturedPost featuredPost={featuredPost} />
+      {/* <SearchBar /> */}
+      <div className="grid grid-cols-1 md:grid-cols-3">
```

Comment:
Since we disabled the SearchBar, should we add some space between this? it feels a little bit crowded here. 'mt-8' would work give enough space on line 32!

Reply:
I'm going to leave the change for another ticket that can tackle additional spacing issues